templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+  <li><a href="{% url 'taxi:index' %}">Home page</a></li>

```

Comment:
Please review the task checklist again: 

 Write realization of sidebar include that must have a list of empty links:
    * Home page
    * Manufacturers
    * Cars
    * Drivers